### PR TITLE
python3Packages: init shap at 0.36.0

### PR DESCRIPTION
**Pull request by evax**
###### Motivation for this change

A unified approach to explain the output of any machine learning model

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/shap/default.nix [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner = "slundberg";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1wxnxvbz6avzzfqjfbcqd4v879hvpq4021v31fhdpccr2q317rr9";
+  };
+
+  # tests are accessing the network
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/shap/default.nix:26]: Do all of them use network or can we just disable the one or three that fail?

File: pkgs/development/python-modules/slicer/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  pname = "slicer";
+  version = "0.0.4";
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "21d53aac4e78c93fd83c0fd2f8f9d8a2195ac079dffdc0da81cd749da0f2f355";
+  };
+
+  checkInputs = [ pytestCheckHook pandas pytorch ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/slicer/default.nix:20]: tests?

File: pkgs/development/python-modules/shap/default.nix (hunk 1 above)
Comment (evax) [reply pkgs/development/python-modules/shap/default.nix]: @SuperSandro2000 there were tests accessing the network, but also an issue with the C extension not being found during tests, all fixed now, thanks for insisting :)

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102729` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python37Packages.shap</li>
    <li>python37Packages.slicer</li>
    <li>python38Packages.shap</li>
    <li>python38Packages.slicer</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102729` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>python38Packages.shap</li>
  </ul>
</details>
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python37Packages.shap</li>
    <li>python37Packages.slicer</li>
    <li>python38Packages.slicer</li>
  </ul>
</details>

**Pull request merged**